sdk/script/management-sdk.ts

```diff
@@ -169,6 +169,11 @@ class AccountManager {
         return this.post(urlEncode `/apps/${appName}/deployments/`, JSON.stringify(deployment), /*expectResponseBody=*/ true)
             .then((res: JsonResponse) => res.body.deployment);
     }
+    
+    public clearDeployment(appName: string, deploymentName: string): Promise<void> {
```

Comment:
I think I will rename all the code methods to be renamed to `clearDeploymentHistory` because it also has parity with the server endpoint being a DELETE on the `/history` route.

And then I will leave the CLI command as `deployment clear`. I think it is just shorter than `deployment clear-history`, and with the CLI helper instructions being printed out, I don't think anyone will be confused.

 Is this an okay compromise?